yorkie/backend/pubsub/mempubsub/mempubsub.go

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2020 The Yorkie Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package mempubsub
+
+import (
+	"sync"
+
+	"github.com/yorkie-team/yorkie/pkg/document/time"
+	"github.com/yorkie-team/yorkie/pkg/log"
+	"github.com/yorkie-team/yorkie/pkg/types"
+	"github.com/yorkie-team/yorkie/yorkie/backend/pubsub"
+)
+
+// Subscriptions is a collection of subscriptions that subscribe to a specific
+// topic.
+type Subscriptions struct {
```

Comment:
The `Subscriptions` doesn't seem to need to be known from the external package. So I think it would be okay to replace it with `subscriptions`.
Could you please tell me why you declared `Subscriptions` as a public structure?

Reply:
Thank you. I think it would be better to change it.